packages/core/directives/map.ts

```diff
@@ -577,6 +583,12 @@ export class AgmMap implements OnChanges, OnInit, OnDestroy {
     this._observableSubscriptions.push(s);
   }
 
+  private _handleTilesLoadedEvent() {
+    const s = this._mapsWrapper.subscribeToMapEvent<void>('tilesloaded').subscribe(
+      () => { this.tilesLoaded.emit(void 0); });
```

Comment:
since this is one line, drop the brackets and semicolon

Reply:
I'd probably do the same as you suggest, but it seems that it is a convention used in other event handlers in AgmMap compontent. Hence, to keep it stylistically consistent, I'd probably leave it the way it is now. Still, it is your call to merge it, so if you're sure we want to change it, then I will do it. What do you think?

Reply:
just cause it was wrong before, doesn't mean it has to be wrong now ;)

Reply:
P.S. no need to do force pushes. Either way, the final merge will be squashed. Up to you.

Reply:
It is technically correct, so I wouldn't say it was wrong - just a little bit verbose. Anyway, it is changed now :)